### ISSUE TITLE
Feature extraction procedure

### ISSUE THREAD

**Issue by yohann84L** [yhu01/BMS#1]
Hey, great job on your paper! 
I was wondering about the extracted features you give (btw you should change in your readme `You can download pre-trained models and extracted features here:` to `You can download extracted features here:`), could you give some details about the training of WRN? Which version of WRN, optimizer, augmentation etc...

Also, did you try another backbone or pretrained method for backbone? It seems for example SSL method such as dino combined with ViT does a great job as feature extraction, it could increase even more the accuracy.

Thanks again
